### PR TITLE
feat(translations): add danish translations

### PR DESCRIPTION
**Pull request by mykz**
## Description

Added Danish (da-DK) translations for Beta

- [x] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

## Type of change

- [ ] Chore (non-breaking change which does not add functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Change to the [templates](https://github.com/payloadcms/payload/tree/main/templates) directory (does not affect core functionality)
- [ ] Change to the [examples](https://github.com/payloadcms/payload/tree/main/examples) directory (does not affect core functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/translations/src/types.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ type DateFNSKeys =
   | 'az'
   | 'bg'
   | 'cs'
+  | 'da-DK'
```
Comment (AlessioGr) [inline packages/translations/src/types.ts:12]: needs to be 'da'

File: packages/translations/src/languages/da.ts [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+    versions: 'Versioner',
+    viewingVersion: 'Se versionen for {{entityLabel}} {{documentTitle}}',
+    viewingVersionGlobal: 'Se version for det globale {{entityLabel}}',
+    viewingVersions: 'Se versioner for {{entityLabel}} {{documentTitle}}',
+    viewingVersionsGlobal: 'Se versioner for det global {{entityLabel}}',
+  },
+}
+
+export const da: Language = {
+  dateFNSKey: 'da-DK',
```
Comment (AlessioGr) [inline packages/translations/src/languages/da.ts:441]: needs to be 'da'

**Review by AlessioGr — CHANGES_REQUESTED** [review]

**Comment by mykz** [comment]
Thanks for the review, @AlessioGr. I've made the corrections, so hopefully, it's all good now. 🤞

**Review by AlessioGr — APPROVED** [review]

**Comment by mykz** [comment]
@AlessioGr Any ideas when this PR might get merged?

**Pull request merged**